lib/workers/package-file/dep-type.js

```diff
@@ -104,6 +106,14 @@ function getDepConfig(depTypeConfig, dep) {
           }
         }
       }
+      if (matchVersion) {
+        depConfig.packageVersionMatches = false;
+        matchVersion.forEach(mVer => {
+          if (isValidSemver(mVer) && matchesSemver(dep.currentVersion, mVer)) {
+            depConfig.packageVersionMatches = true;
```

Comment:
Instead of adding a new variable/option here, we should just not apply the package rule